src/network/tcp.py

```diff
@@ -231,37 +242,55 @@ def handle_close(self):
 
 
 class Socks5BMConnection(Socks5Connection, TCPConnection):
+    """TBC"""
+
     def __init__(self, address):
         Socks5Connection.__init__(self, address=address)
         TCPConnection.__init__(self, address=address, sock=self.socket)
         self.set_state("init")
 
     def state_proxy_handshake_done(self):
+        """TBC"""
         Socks5Connection.state_proxy_handshake_done(self)
         self.nodeid = randomBytes(8)
-        self.append_write_buf(protocol.assembleVersionMessage(self.destination.host, self.destination.port, \
-                network.connectionpool.BMConnectionPool().streams, False, nodeid=self.nodeid))
+        self.append_write_buf(
+            protocol.assembleVersionMessage(
+                self.destination.host,
+                self.destination.port,
+                network.connectionpool.BMConnectionPool().streams,
+                False,
+                nodeid=self.nodeid))
         self.set_state("bm_header", expectBytes=protocol.Header.size)
         return True
 
 
 class Socks4aBMConnection(Socks4aConnection, TCPConnection):
+    """TBC"""
+
     def __init__(self, address):
         Socks4aConnection.__init__(self, address=address)
         TCPConnection.__init__(self, address=address, sock=self.socket)
         self.set_state("init")
 
     def state_proxy_handshake_done(self):
+        """TBC"""
```

Comment:
State when SOCKS4a connection succeeds, we need to send a Bitmessage handshake to peer.